docs/dlk/archive_file.md

```diff
@@ -12,9 +12,9 @@ make ar_fpga
 
 These commands will generate the following files:
 ```
-libdlk_x86.a
-libdlk_arm.a
-libdlk_fpga.a
+lib_x86.a
```

Comment:
In #724, @primenumber change library name. `libdlk_x86` is correct.

Reply:
Oh. Thanks for pointing that out. I'll revert these changes.